migration/migrating-3-4/migrating-openshift-3-to-4.adoc

```diff
@@ -1,7 +1,8 @@
 [id="migrating-openshift-3-to-4"]
 = Migrating {product-title} 3.x to 4.2
 include::modules/common-attributes.adoc[]
-:context: migrating-openshift-3-to-4
+:context: ocp-3-to-4
```

Comment:
Is there a reason this context was changed? Typically we try not to change anchors once something has been published.

Reply:
The context was too long and it seemed odd to see "migrating" so frequently in links. I think it's early enough in the publication process that changing the context won't be a problem. I'll keep it in mind for the future.